discovery-provider/plugins/network-monitoring/src/db/migrations/create_tables.ts

```diff
@@ -95,11 +99,11 @@ const migration: RunnableMigration<QueryInterface> = {
     },
     down: async (params: MigrationParams<QueryInterface>) => {
         await params.context.sequelize.query(`
-            DROP TABLE network_monitoring_index_blocks;
-            DROP TABLE network_monitoring_content_nodes;
-            DROP TABLE network_monitoring_users;
-            DROP TABLE network_monitoring_cids_from_discovery;
             DROP TABLE network_monitoring_cids_from_content;
+            DROP TABLE network_monitoring_cids_from_discovery;
+            DROP TABLE network_monitoring_users;
+            DROP TABLE network_monitoring_content_nodes;
+            DROP TABLE network_monitoring_index_blocks;
```

Comment:
I realized that it wouldn't work originally because of constraints - we don't use this way of reversing the migrations anywhere in the code but it seems like best practice to have it here.